### ISSUE TITLE
Infill line multiplier + (dis)connect layer lines [Already in Cura since 2018]

### ISSUE THREAD

**Issue by hutje** [SoftFever/OrcaSlicer#3650]
**Infill line multiplier**

As described in this issue on BambuStudio Github: [https://github.com/bambulab/BambuStudio/issues/1528#issuecomment-1640671376](https://github.com/bambulab/BambuStudio/issues/1528#issuecomment-1640671376)

![IMG_8306](https://github.com/SoftFever/OrcaSlicer/assets/15308350/1433d2d5-4803-42e4-849e-adb2f530a622)
![IMG_8307](https://github.com/SoftFever/OrcaSlicer/assets/15308350/f60cd0ed-8886-4720-8a8b-867d3a2a318a)

It would be really useful to be able to have the option to have a multiplier value for the infill lines. Sometimes you don't want to increase the infill percentage but still want a stronger infill.

**Connect infill lines**
![IMG_8309](https://github.com/SoftFever/OrcaSlicer/assets/15308350/cd872b09-5240-435a-88b5-ebb2a830af82)
![IMG_8308](https://github.com/SoftFever/OrcaSlicer/assets/15308350/8681b5e3-a39e-4501-802b-0406863c3e7b)

Another addition to this would be to be able to have a box to enable and disable "connect layer lines". This way you can expose infill and keep it nice looking. As gyroid connects the layerlines at walls. But without walls, it doesn't look good.
As described here:
https://github.com/Ghostkeeper/SettingsGuide/blob/master/resources/articles/infill/zig_zaggify_infill.md#connect-infill-lines
****

Both these features are already in Cura. It would be nice to have this in OrcaSlicer as well.
This would also make nice designs like this: 
![IMG_8315](https://github.com/SoftFever/OrcaSlicer/assets/15308350/e63d64eb-84e2-45d2-936a-a592dd1fa4f1)

**Comment by igiannakas** [comment]
You can increase line width - for a 0.4 nozzle you can go easily to 0.7-0.8 which will give you better/ stronfer results than simply extruding two lines. 

CNC kitchen has done a video on it.

**Comment by igiannakas** [comment]
https://youtu.be/9YaJ0wSKKHA

**Comment by hutje** [comment]
> You can increase line width - for a 0.4 nozzle you can go easily to 0.7-0.8 which will give you better/ stronfer results than simply extruding two lines.
> 
> CNC kitchen has done a video on it.

True, but not all filament would be suitable to do this. As there is more backpressure when extruding. When using soft TPU I would prefer multiple lines.

**Comment by hutje** [comment]
Still hope this option will be implemented... It is not an entirely new feature...

**Comment by Sureiya** [comment]
I also would really like this feature.

**Comment by hutje** [comment]
I don't understand why this feature is not been implemented already. This feature is already available in cura. It is really useful for printing big objects or using flexible filaments.

**Comment by RF47** [comment]
This feature is useful, please add.

**Comment by ShaneDelmore** [comment]
> > You can increase line width - for a 0.4 nozzle you can go easily to 0.7-0.8 which will give you better/ stronfer results than simply extruding two lines.
> > CNC kitchen has done a video on it.
> 
> True, but not all filament would be suitable to do this. As there is more backpressure when extruding. When using soft TPU I would prefer multiple lines.

I don't think backpressure is an issue if you have correctly set the maximum flow rate for the filament, it will then slow down as needed to the correct speed to ensure the flow rate is not exceeded.

**Comment by hutje** [comment]
> > > You can increase line width - for a 0.4 nozzle you can go easily to 0.7-0.8 which will give you better/ stronfer results than simply extruding two lines.
> > > CNC kitchen has done a video on it.
> > 
> > 
> > True, but not all filament would be suitable to do this. As there is more backpressure when extruding. When using soft TPU I would prefer multiple lines.
> 
> I don't think backpressure is an issue if you have correctly set the maximum flow rate for the filament, it will then slow down as needed to the correct speed to ensure the flow rate is not exceeded.

I am not asking for reasons why you don't think it is needed.
The fact you don't need the feature, doesn't mean other people do find it useful to be able to use multiple infill lines.

**Comment by ShaneDelmore** [comment]
I didn't intend it as a reason why the feature is not needed.  You keep on trying to get people to do free work for you with that winning personality.
